torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -587,7 +597,36 @@ void ComputeSubgraphInMKLDNN(Node* subgraph_node) {
     }
 
     if (body_node->kind() == aten::relu6) {
-      body_node->replaceWithNewSymbol(prim::MKLDNNRelu6);
+      WithInsertPoint insert_guard{body_node};
+      auto out_node = body_node->owningGraph()->create(
+          {prim::MKLDNNHardTanh}, {body_node->input(0)}, 1);
+      out_node->f_(attr::min_val, 0.);
+      out_node->f_(attr::max_val, 6.);
+      body_node->owningGraph()->insertNode(out_node);
+      auto out_val = out_node->output();
+      out_val->copyMetadata(body_node->output());
+      body_node->output()->replaceAllUsesWith(out_val);
+      body_node->destroy();
+      continue;
+    }
+
+    if (body_node->kind() == aten::hardtanh) {
+      WithInsertPoint insert_guard{body_node};
```

Comment:
bump here, I think this would be more easily handled if we just had inputs instead of attributes